src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -413,14 +414,44 @@ public static void updateGitSourceUrl(BuildConfig buildConfig, String gitUrl, St
     gitSource.setRef(ref);
   }
 
+  public static void updateOpenShiftBuildPhaseAndTriggerMessage(Build build, String phase, String message) {
```

Comment:
Just call this `updateOpenShiftBuildPhase` ... Java will allow the 2 versions with one having a super set param list of the other

Reply:
ok, sounds good